cpp/models/ide_secir/model.cpp

```diff
@@ -22,7 +22,6 @@
 #include "infection_state.h"
 #include "memilio/config.h"
 #include "memilio/utils/logging.h"
-#include "memilio/math/eigen.h"
```

Comment:
Because clangd said it is not used directly, I guess it was already included somewhere else.

Reply:
I guess it nevertheless makes sense to add the include here as it is used. If it is included somewhere else and removed there, it is not observable that this may cause problems somewhere else.